features/glance.feature

```diff
@@ -0,0 +1,39 @@
+@glance
+Feature: Glance
+  As an administrator
+  I want to make sure that Glance component is working correctly
+
+  @create_kvm_image
```

Comment:
It's for convenience, it's useful when writing and testing a scenario and it's useful when you want to list scenarios for a specific feature with `rake` command (no need to look for the feature file and read the text). 

This is easy to drop when they prove to cause more overhead than benefit later.